price_parser/parser.py

```diff
@@ -283,3 +292,31 @@ def parse_number(num: str) -> Optional[Decimal]:
         return Decimal(num)
     except InvalidOperation:
         return None
+
+
+def date_format(price):
+    for fmt in ['%d.%m.%Y', '%B, %Y', '%b, %Y', '%Y-%m-%d']:
+        try:
+            date = datetime.strptime(price, fmt)
+            if isinstance(date, datetime):
```

Comment:
If I’m reading the coverage data correctly, I think tests do not cover cases where this evaluates to `False`.

Reply:
Many thanks for the suggestions @Gallaecio. Fixed in https://github.com/scrapinghub/price-parser/pull/19/commits/20ac018ad2b57a9316da239d38e1feda69661735